### DISCUSSION TITLE
About ingester resource overhead

### DISCUSSION THREAD

**Discussion by kwenzh** [grafana/mimir#3416] [Help and support] [answered]
prometheus has 2 pod, keep 6h data , resource 

![image](https://user-images.githubusercontent.com/38367518/200712438-dbc39569-655d-4e42-b584-65c2fd39e951.png)

ingester

![image](https://user-images.githubusercontent.com/38367518/200711740-c82c4612-51d1-462a-b8df-941f2db691e2.png)

That adds up to quite a bit of ingester cost memory consumption, okay?

**Comment by Logiraptor** [comment, marked as answer]
Hello :wave:, please take a look at this documentation for some estimates about expected memory usage: https://grafana.com/docs/mimir/latest/operators-guide/run-production-environment/planning-capacity/#ingester

Does that help?

**Reply by kwenzh** [reply]
> Hello wave, please take a look at this documentation for some estimates about expected memory usage: https://grafana.com/docs/mimir/latest/operators-guide/run-production-environment/planning-capacity/#ingester
> 
> Does that help?

ok ！ thank you ， sum(prometheus_tsdb_head_series) is 1800 0000 , and ingster  need cost 300GB memory

**Reply by pracucci** [reply]
If you have 1.8M in-memory series in Prometheus, storing these series in Mimir will require about 45GB of ingesters memory according to the capacity planning doc:

> Memory: 2.5GB for every 300,000 series in memory

Where the actual number of series in Mimir must be multiplied by 3x due to the replication factor. Typically the capacity planning doc is a bit pessimistic, so the actual memory utilization may be lower.
